### DISCUSSION TITLE
osnadmin binary not included in last release

### DISCUSSION THREAD

**Discussion by iasinDev** [hyperledger/fabric#2507] [General]
Hi, by the documentation, https://hyperledgendary.github.io/unstable-fabric-docs/install.html

I understood that in the binaries zip would be all binaries, but "osnadmin" is not included in the last release zip https://github.com/hyperledger/fabric/releases/tag/v1.4.11

It's normal, is an error, a mismatch between doc and code ... ?

Thanks in advance

**Comment by lindluni** [comment]
osnadmin was not part of the 1.4 stream, it was introduced in 2.3 and later as it is used to manage APIs that didn't exist in 1.4.

**Reply by iasinDev** [reply]
ahh ok ok, thanks for the clarification!
